package com.patterns;

import java.util.Scanner;

/*5
        * * * * *
         * * * *
          * * *
           * *
            *
           * *
          * * *
         * * * *
        * * * * *

 */

public class HourGlass {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int length = sc.nextInt();

        for(int i=1; i<=length; i++){
            for(int j=1; j<i; j++)
                System.out.print(" ");
            for(int j=1; j<=length-i+1; j++)
                System.out.print("* ");

            System.out.println();
        }

        for(int i=length-1; i>=1; i--){
            for(int j=1; j<i; j++)
                System.out.print(" ");
            for(int j=1; j<=length-i+1; j++)
                System.out.print("* ");

            System.out.println();
        }
    }
}
